docs/V3/reference/error-codes.md

```diff
@@ -0,0 +1,51 @@
+---
+id: error-codes
+title: Error Codes
+---
+
+
+LiquidityMath.sol
+
+* `LS` : Liquidity Sub
+* `LA`: Liquidity Add
+
+Oracle.sol
+
+* `OLD`: The target must be chronologically at or after the oldest observation
+
+* `I`: Cardinality must be greater than 0
+  
+Position.sol
+
+* `NP`: disallow pokes for 0 liquidity positions
+
+Tick.sol
+
+* `LO` LiquidityGrossAfter must be less than MaxLiquidity
+
+TickMath.sol 
+
+* `T`:  require(absTick <= uint256(MAX_TICK), 'T');
+* `R`: second inequality must be < because the price can never reach the price at the max tick
+
+TransferHelper.sol
+
+* `TF`: Transfer Failed : errors with TF if transfer fails
+
+
+UniswapV3Pool.sol
+
+* `LOK`: The reentrancy guard. Each pool can only be called once during a single transaction. This includes multiple swaps, or withdrawing a flash and then swapping on the same pool afterwards 
```

Comment:
a pool can be called multiple times in a single transaction, you just cannot re-enter the pool mid-swap (and multiple swaps/mints/burns can all happen in 1 tx)